encrypted-media-respec.html

```diff
@@ -3396,9 +3413,11 @@
           <ul>
             <li><p><a def-id="distinctive-identifier">Distinctive Identifier(s)</a></p></li>
             <li><p>Origins visisted (via stored or in-memory data, permissions, etc.)</p></li>
-            <li><p>Content viewed (via stored or in-memory keys, key IDs, data, etc.)</p></li>
+            <li><p>Content viewed (via stored or in-memory licenses, keys, key IDs, records of key usage, records of license destruction, etc.)</p></li>
           </ul>
           <p>This specification presents a specific concern because such information is commonly stored outside the user agent (and its profile storage), often in the CDM.</p>
+          
+          <p>Since the content of licenses are key-system specific and since key IDs are arbitrary, these data items could be abused to store user-identifying information.</p>
```

Comment:
Or was this your recommendation?

> Since the content of licenses, records of key usage, and records of license destruction are Key System-specific and since key IDs may contain any value, these data items could be abused to store user-identifying information.